Sources/PrimerSDK/Classes/Core/Analytics/AnalyticsStorage.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  AnalyticsStorage.swift
+//  PrimerSDK
+//
+//  Created by Jack Newcombe on 01/12/2023.
+//
+
+import Foundation
+
+private let analyticsURL: URL = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)[0].appendingPathComponent("analytics")
```

Comment:
nit-pick: Maybe change the name to indicate that this is a local storage URL... Is this fileprivate or private?

Reply:
Good shout! ❤️